content/en/blog/2023/end-user-discussions-04.md

```diff
@@ -0,0 +1,111 @@
+for users in the Asia-Pacific (APAC) region.
+
+The discussions take place using a
+[Lean Coffee format](https://agilecoffee.com/leancoffee/), whereby folks are
+invited to post their topics to the
+[Agile Coffee board like this one](http://agile.coffee/#2f83c1c1-918c-4c78-8671-194b2e9d8e54),
+and everyone in attendance votes on what they want to talk about.
+
+## What we talked about
+
+We talked about evangelizing the adoption of OpenTelemetry in a big organization
+and also discussed how to optimize observability data at scale.
+
+## Discussion Highlights
+
+Below is the summary of this month's discussion.
+
+### Evangelizing adoption of OpenTelemetry in a big organization
+
+**Q:** How do you evangelize the adoption of OpenTelemetry in a big
+organization?
+
+**A:** In a big organization, the first step would be to put out the current
+pain points of observability to leadership. There are benefits of having an open
+source standard for observability. If there is no standard in place, it gets
+very difficult to communicate across different teams. If you use OpenTelemetry,
+you do not have to depend on any vendor agents, and you have the flexibility to
+send data to multiple backends.
+
+### How to optimize observability data at scale?
+
+**Q:** In a big organization, observability data can be in the range of TBs per
+day, which comes with associated costs. But there is always a feeling that 80%
+of captured data is unusable. None of the vendors help you understand what data
+is accessed and how to bring that visibility to the engineering teams sending
+the data.
+
+**A:** One of the ways to optimize observability at scale is sampling. There are
+a number of options for you to reduce the data volumes at the SDKs level and the
+collector level.
+
+Also, there is active work going on the OpenTelemetry Collector side to handle
+data at scale more efficiently. For example, there is work going around using
+Apache Arrow for serialization to optimize network costs.
+
+One of the other ways is to look at retention settings for the collected
+telemetry data. There is no need to retain data that won't be used later.
+
+## Other Important discussion points
+
+#### Maturity model for OpenTelemetry
+
+**Q:** Is there some literature available around understanding steps to reach a
+certain maturity level in adopting OTel in your organization? For example, I
+should be able to go to a team and tell them to start with X, and then do Y to
+move ahead. In a big enterprise, you have to provide something for people to
+understand the maturity of OpenTelemetry.
+
+**A:** OpenTelemetry metrics is a good place to start for infrastructure
+monitoring. For teams adopting OpenTelemetry, a good idea is to start with
+minimal changes. For example, teams can start with languages that have
+auto-instrumentation support. Seeing value from small changes can build more
+confidence in the team to go deeper into OpenTelemetry adoption.
```

Comment:
I have my issues with this answer: from an otel perspective infrastructure metrics are an odd place to start, this is something you can do with prometheus or any other metrics-focused solution. I agree with the latter part that using auto instrumentation first to get traces (and metrics, and logs), and then dig deeper.

Reply:
I agree re: the metrics comment. I'd prefer to leave that out altogether.

Reply:
@svrnm @avillela done. So this comment was mentioned by an OTel user, he must be using receivers to start with metrics monitoring.